c2_utils/src/mfx_va_allocator.cpp

```diff
@@ -603,7 +605,10 @@ mfxStatus MfxVaFrameAllocator::CreateSurfaceFromGralloc(const IMfxGrallocModule:
     desc.num_objects = 1;
     desc.objects[0].fd = info.prime;
     desc.objects[0].size = decode_target ? info.pitches[0] * ((height + 31) & ~31) * 1.5 : info.pitches[0] * ((height + 15) & ~15) * 1.5;
-    desc.objects[0].drm_format_modifier = I915_FORMAT_MOD_Y_TILED;
```

Comment:
Why setting to I915_FORMAT_MOD_Y_TILED by default was good, but now is an issue? What is the cause of this issue?

Reply:
Hi Yuanjun,

This scrcpy issue is a regression issue caused by this newly merged PR https://github.com/projectceladon/MediaSDK_C2/pull/99, in which we changed the API to create surface (for future use in decoding on dGPU). With PR99, the input surface in RGB format is forced to use tile format to do encoding, but actually it is created by virtual display without tile information. Our fix is to explicitly add tile information according to the exact surface format.